plugins/push/frontend/public/templates/mobile-message-preview.html

```diff
@@ -0,0 +1,58 @@
+<div class="mobile-preview">
+    <div class="mobile-preview__images">
+        <img v-if="isIOSPlatformSelected()" src="/images/push/preview.i.png" />
+        <img v-else-if="isAndroidPlatformSelected()" src="/images/push/preview.a.png" />
+    </div>
+
+    <div v-if="isIOSPlatformSelected()" class="mobile-preview__ios-message">
+        <div class="mobile-preview__ios-message__header">
+            <img class="mobile-preview__ios-message__header__image" src="/appimages/5ea4bd674bb4a33298a81727.png" />
```

Comment:
i think we had a small discussion about nesting in BEM.
we should try to avoid it.
https://en.bem.info/methodology/quick-start/#guidelines-for-using-elements

Reply:
we should try to keep class names flat.

Reply:
yes, I will revert to flat class style (by the time we decided to go down that road, I had already implemented it).